### PR TITLE
creating atleast_2d function

### PR DESCRIPTION
**Pull request by ndurumo254**
close https://github.com/unifyai/ivy/issues/10903

### CODE REVIEW

**Comment by ndurumo254** [comment]
#8422

**Comment by ndurumo254** [comment]
#https://github.com/unifyai/ivy/pull/10905#issuecomment-1442434800

File: ivy/functional/frontends/jax/numpy/manipulations.py [hunk 1]
```diff
@@ -88,14 +88,14 @@ def stack(arrays, axis=0, out=None, dtype=None):
...
 def take(
-    a,
-    indices,
-    axis=None,
-    out=None,
-    mode=None,
-    unique_indices=False,
-    indices_are_sorted=False,
-    fill_value=None,
+        a,
```
Comment (hirwa-nshuti) [inline ivy/functional/frontends/jax/numpy/manipulations.py:91]: These added spaces are not needed

**Review by hirwa-nshuti — CHANGES_REQUESTED** [review]
Hey @ndurumo254 

The implementation looks good but you have also changed many other functions where there are added unnecessary spaces would you remove them.

After that is resolved I'll merge your PR

Thanks😊

**Comment by hirwa-nshuti** [comment]
Hey @ndurumo254 
Still there are many unnecessary changes if you head here you can find them
https://github.com/unifyai/ivy/pull/10980/files

File: ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_manipulation.py [hunk 2]
```diff
@@ -59,12 +59,12 @@ def _get_clip_inputs(draw):
...
 def test_jax_numpy_clip(
-    *,
-    input_and_ranges,
-    on_device,
-    fn_tree,
-    frontend,
-    test_flags,
+        *,
+        input_and_ranges,
+        on_device,
```
Comment (hirwa-nshuti) [inline ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_manipulation.py:64]: These extra tabs aren't supposed to be here.

**Review by hirwa-nshuti — CHANGES_REQUESTED** [review]
Hey @ndurumo254 

There are too many formatting errors in your PR😊
You can use the pre-commit hook to remove them here is a guide to it https://lets-unify.ai/ivy/contributing/setting_up.html#pre-commit

After successfully installing it you can run `pre-commit run --files FILE_PATH` for both the changed files it will reformat the code automatically and push the changes.

Hope this will help out🙂

Thanks

**Comment by ndurumo254** [comment]
> Hey @ndurumo254
> 
> There are too many formatting errors in your PR😊 You can use the pre-commit hook to remove them here is a guide to it https://lets-unify.ai/ivy/contributing/setting_up.html#pre-commit
> 
> After successfully installing it you can run `pre-commit run --files FILE_PATH` for both the changed files it will reformat the code automatically and push the changes.
> 
> Hope this will help out🙂
> 
> Thanks

done

**Review by hirwa-nshuti — CHANGES_REQUESTED** [review]
I checked from the frontends and this function is already implemented there as I can see is that what you did change its location in the same files😊

https://github.com/unifyai/ivy/blob/37a5139bdee81f49a84ad63aa9d6597bc0c0b4f8/ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_manipulation.py#L663

Would you work on another task
And mind checking if the function exists before spending much time re-implementing the same stuff🙂

Thanks

**Comment by ndurumo254** [comment]
> I checked from the frontends and this function is already implemented there as I can see is that what you did change its location in the same files😊
> 
> https://github.com/unifyai/ivy/blob/37a5139bdee81f49a84ad63aa9d6597bc0c0b4f8/ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_manipulation.py#L663
> 
> Would you work on another task And mind checking if the function exists before spending much time re-implementing the same stuff🙂
> 
> Thanks

ok.

**Pull request closed without merging**